### DISCUSSION TITLE
Migration tool for `octavia-cli` to Terraform script

### DISCUSSION THREAD

**Discussion by marcosmarxm** [airbytehq/airbyte#31286] [Connector Ideas and Features]
As a user, I have been using the Octavia CLI to manage my connectors and connections as code. Airbyte is deprecating the `octavia-cli` in favor of using Terraform, and I need a tool to facilitate the migration.

**Comment by tlatrace** [comment]
This octavia-to-terraform migration tool would indeed be super helpful. As octavia users, we won't be able to upgrade to the latest Airbyte version until we create some Terraform resources for our sources/destinations/connections (we want to continue to manage our Airbyte resources with IaC)
